eng/dnx/nuspec/README.md

```diff
@@ -0,0 +1,430 @@
+# Azure MCP Server .NET Tool
+
+- Install the Azure MCP Server .NET tool from NuGet to add Model Context Protocol (MCP) capabilities to your Azure projects and enable integration with VS Code.
+
+## Table of Contents
+- [Overview](#overview)
+- [Getting Started](#getting-started)
+- [What can you do with the Azure MCP Server?](#what-can-you-do-with-the-azure-mcp-server)
+- [Complete List of Supported Azure Services](#complete-list-of-supported-azure-services)
+- [Documentation](#documentation)
+- [Feedback & Support](#feedback--support)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Overview
+
+**Azure MCP Server** adds smart, context-aware AI tools right inside VS Code to help you work more efficiently with Azure resources. The Azure MCP Server supercharges your agents with Azure context across **30+ different Azure services**.
+
+## Getting Started
+
+### Requirements
+To run the Azure MCP server, you must have [.NET 10 Preview 6 or later](https://dotnet.microsoft.com/en-us/download/dotnet/10.0) installed. This version of .NET adds a command, dnx, which is used to download, install, and run the MCP server from [nuget.org](https://www.nuget.org).
+
+To verify your .NET version, run the following command in your terminal:
+
+```
+dotnet --info
+```
+
+### Configuration
+To configure the MCP server for use with VS Code, use the following snippet and include it in your `mcp.json`
+```
+"servers": {
+	"Azure MCP Server": {
+		"command": "dnx",
+		"args": [
+			"Azure.Mcp",
+			"--source",
+			"https://api.nuget.org/v3/index.json",
+            "--yes",
+			"--",
+			"azmcp",
+			"server",
+			"start"
+		],
+		"type": "stdio"
+	}
+}
+```
+If you'd like to use a specific version of the Azure MCP server, you can specify it with the --version argument, like so:
+```
+"servers": {
+	"Azure MCP Server": {
+		"command": "dnx",
+		"args": [
+			"Azure.Mcp",
+			"--source",
+			"https://api.nuget.org/v3/index.json",
+			"--version",
+			"0.6.0",
+			"--yes",
+			"--",
+			"azmcp",
+			"server",
+			"start"
+		],
+		"type": "stdio"
+	}
+}
+```
+When configured this way, you will need to update the version as new release become available.
+
```

Comment:
Do we also need to add the notes to start / stop the server? I am doing that for npm one

Reply:
Seems more than is required on the nuget page.
Perhaps this could be added to the main project README.md